build.rs

```diff
@@ -39,17 +48,28 @@ fn main() {
     if !use_bzip2 {
         let bzip2_patterns = vec!["s/ -lbz2//", "/#define HAVE_LIBBZ2/d"];
         sed_htslib_makefile(&out, &bzip2_patterns, "bzip2");
+    } else if let Ok(inc) = env::var("DEP_BZIP2_ROOT")
```

Comment:
Why are these additional env vars needed?

Reply:
Do they come from the bzip2-sys crate?

Reply:
that's so sad I mistakenly deleted the commonts :cry: , maybe you could see them in email?